fdbclient/include/fdbclient/StorageServerShard.h

```diff
@@ -31,12 +31,22 @@ struct StorageServerShard {
 
 	enum ShardState {
 		NotAssigned = 0,
-		MovingIn = 1,
+		Adding = 1,
```

Comment:
Maybe comment on the meaning of these states.

Reply:
Done.